src/index.spec.ts

```diff
@@ -379,7 +382,7 @@ describe('EppoJSClient E2E test', () => {
       flags: {
         [hashedFlagKey]: mockExperimentConfig,
       },
-    };
+    } as unknown as Record<string, IExperimentConfiguration>;
```

Comment:
TypeScript insisted on the double `as` because we are wiling out with our types here.